sysadmin-ssh-keys/README.md

```diff
@@ -1,2 +0,0 @@
-This directory contains the SSH keys used for the sysadmin user.
```

Comment:
We need this directory to be present as it stores the keys. Please add this directory back